vaadin-upload-flow-parent/vaadin-upload-flow-integration-tests/src/test/java/com/vaadin/flow/component/upload/tests/AbstractUploadIT.java

```diff
@@ -9,11 +9,25 @@
 
 public abstract class AbstractUploadIT extends AbstractComponentIT {
     /**
+     * Creates a temp file with ".txt" extension for testing purposes.
+     *
      * @return The generated temp file handle
      * @throws IOException
      */
     File createTempFile() throws IOException {
```

Comment:
I would suggest that we always require tests to specify the extension when creating temp files. That will be more explicit. In other words, I suggest only leaving `createTempFile(String extension)`.

Reply:
That makes sense, considering extension is important for file upload. Updated the PR.